script/hatchet.ini

```diff
@@ -1,73 +1,77 @@
 [run]
 # What individual steps of HATCHet should we run in the pipeline?
 # Valid values are True or False
-download_panel=True
```

Comment:
A lot of these changes are just introduction of whitespace to increase readibility.